test/transforms/cypress-plugins.test.js

```diff
@@ -1,8 +1,14 @@
 import expect from 'expect';
 import applyTransform, { dedent } from '../helpers/apply-transform';
 import transform from '../../transforms/cypress-plugins';
+import setupCodeshift from '../helpers/setup-codeshift';
```

Comment:
What if `setup-codeshift` executed itself and we utilize mocha's `--require` to make sure it runs before all tests?